app/retail/views.py

```diff
@@ -496,6 +504,70 @@ def browser_extension_firefox(request):
     return redirect('https://addons.mozilla.org/en-US/firefox/addon/gitcoin/')
 
 
+def idea_show(request, idea_id):
+    return TemplateResponse(request, 'idea.html', {})
+
+
+@csrf_exempt
+def new_idea(request):
+    return TemplateResponse(request, 'new_idea.html', {})
+
+
+@require_POST
+@csrf_exempt
+def create_idea(request):
+    idea = Idea(**json.loads(request.body))
```

Comment:
Just a note, we'll ultimately want to migrate this to a form to handle cleaning and validation on the posted values.

Reply:
i get it!